src/components/navigation-header/components/product-page-content/index.tsx

```diff
@@ -1,154 +1,59 @@
-// Third-party imports
-import classNames from 'classnames'
-
 // HashiCorp Imports
-import InlineSvg from '@hashicorp/react-inline-svg'
-import BoundaryLogo from '@hashicorp/mktg-logos/product/boundary/primary-padding/colorwhite.svg?include'
-import ConsulLogo from '@hashicorp/mktg-logos/product/consul/primary-padding/colorwhite.svg?include'
-import HashiCorpLogo from '@hashicorp/mktg-logos/corporate/hashicorp/logomark/white.svg?include'
-import HCPLogo from '@hashicorp/mktg-logos/product/hcp/no-logomark/white.svg?include'
-import NomadLogo from '@hashicorp/mktg-logos/product/nomad/primary-padding/colorwhite.svg?include'
-import PackerLogo from '@hashicorp/mktg-logos/product/packer/primary-padding/colorwhite.svg?include'
-import TerraformLogo from '@hashicorp/mktg-logos/product/terraform/primary-padding/colorwhite.svg?include'
-import VagrantLogo from '@hashicorp/mktg-logos/product/vagrant/primary-padding/colorwhite.svg?include'
-import VaultLogo from '@hashicorp/mktg-logos/product/vault/primary-padding/colorwhite.svg?include'
-import WaypointLogo from '@hashicorp/mktg-logos/product/waypoint/primary-padding/colorwhite.svg?include'
```

Comment:
Nice I wonder if this is where the bundle savings is coming from, loading just the mark instead of the whole logo

Reply:
Possibly! I feel like we're probably already loading `ProductIcon` elsewhere on the page, which might help too 🤔